servers/cromwell/jobs/controllers/jobs_controller.py

```diff
@@ -394,3 +393,18 @@ def _format_query_labels(orig_query_labels):
     for key, val in orig_query_labels.items():
         query_labels[urllib.unquote(key)] = urllib.unquote(val)
     return query_labels
+
+
+def _get_scattered_task_status(metadata):
+    # get all shard statuses
+    statuses = {
+        task_statuses.cromwell_execution_to_api(shard.get('executionStatus'))
+        for shard in metadata
+    }
+    # return status by ranked applicability
+    for status in [
+            'Failed', 'Aborted', 'Aborting', 'Running', 'Submitted',
```

Comment:
Should `Aborted` and `Aborting` switch places? It seems that one wouldn't want to display `Aborted` if some aborts haven't finished

Reply:
done